packages/card/stories/RadioCard.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import type { StoryFn, StoryObj } from '@storybook/react';
+
+import { RadioCard } from '../src/RadioCard';
+
+export default {
+  component: RadioCard,
+  title: 'Components/Card/RadioCard',
+  description: 'A radio button with a label and optional image and subtext.',
+  decorators: [
+    (Story: StoryFn) => (
+      <div style={{ display: 'flex', width: '24rem' }}>
+        <Story />
+      </div>
+    ),
+  ],
+  parameters: {
+    status: {
+      type: import.meta.env.STORYBOOK_PACKAGE_STATUS__RADIOCARD,
+    },
+  },
+};
+
+type Story = StoryObj<typeof RadioCard>;
+
+const commonProps = {
+  label: 'Label',
+};
+
+export const Simple: Story = {
```

Comment:
Clicking on a card seems to do nothing. Should `checked` toggle with that interaction?

Reply:
`checked` only has a visual effect. You would need to use an event handler and set checked to something like `checked={state === id}` for it to toggle.